### PR TITLE
Builds out about section

### PR DESCRIPTION
**Pull request by Mousecop**
PR: https://github.com/BeamLabs/Kingsland-Farm/issues/3

### CODE REVIEW

**Comment by Mousecop** [comment]
Also closes https://github.com/BeamLabs/Kingsland-Farm/issues/4

File: src/pages/home/Home.css [hunk 1]
```diff
@@ -17,4 +17,78 @@
...
+    display: block;
+    width: 100%;
+  }
+
+  .embed-iframe {
+    width: 90%;
+    height: 90%;
+  }
+
+  /*.square-img {
```
Comment (agiron123) [inline src/pages/home/Home.css]: remove comment please if we're not using it any more.

File: src/pages/home/Home.css [hunk 2]
```diff
@@ -17,4 +17,78 @@
...
+hr {
+  width: 80%;
+}
+
+.aboutSection-container {
+  width: 80%;
+  margin: 40px auto 0 auto;
+}
+
+/*.square-img {
```
Comment (agiron123) [inline src/pages/home/Home.css]: Remove comment

**Comment by Mousecop** [comment]
updated branch. Removed comments

**Pull request merged**